drivers/storage/portworx/components_test.go

```diff
@@ -2662,6 +2662,41 @@ func TestAutopilotInvalidCPU(t *testing.T) {
 		fmt.Sprintf("%v %v Failed to setup Autopilot.", v1.EventTypeWarning, util.FailedComponentReason))
 }
 
+func TestSecurityInstall(t *testing.T) {
```

Comment:
Can we add couple more tests here - 
- When there are env variables already present in StorageCluster, ensure secrets don't have auto-generated values, but contain the ones from env variables.
- Disable security and verify that the secret is deleted (What would happen in this case? Can we really change PX to a non-secure mode? If not, do we want to delete the secrets?)

Reply:
Just checked and the secret is not deleted when you disable security.